pkg/f1/testing/t.go

```diff
@@ -27,36 +31,70 @@ type T struct {
 func NewT(iter, scenarioName string) (*T, func()) {
 	t := &T{
 		Iteration:     iter,
-		Log:           log.WithField("i", iter).WithField("scenario", scenarioName).Logger,
+		Logger:        log.WithField("i", iter).WithField("scenario", scenarioName).Logger,
```

Comment:
Do we have to change this? A lot of scenarios will need changing to use the new name.

Reply:
We change it because it was having a conflict with the new `Log()` method.